src/Nethermind/Nethermind.Evm/Instructions/EvmInstructions.cs

```diff
@@ -6,13 +6,12 @@
 using System.Runtime.CompilerServices;
 using Nethermind.Core;
 using Nethermind.Core.Specs;
-using Nethermind.Evm.Precompiles;
 
 namespace Nethermind.Evm;
 using unsafe OpCode = delegate*<VirtualMachine, ref EvmStack, ref long, ref int, EvmExceptionType>;
 using Int256;
 
-internal static unsafe partial class EvmInstructions
+public static unsafe partial class EvmInstructions
```

Comment:
Move stuff to `Nethermind.Evm.Precompiles`. Keep this internal but add `InternalsVisibleTo` attribute.